test/integration/offinstall_int_test.go

```diff
@@ -57,6 +57,14 @@ func (suite *OffInstallIntegrationTestSuite) TestInstallAndUninstall() {
 		defer func() {
 			suite.Require().NoError(env.Set("PATH", origPath))
 		}()
+	} else {
```

Comment:
Now that you're cleaning up on all platforms, would you please change the above comment to something like "Clean up env after test"?